### ISSUE TITLE
[A11y] Desktop Advanced Search: Error Select-name for All or Any

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1741]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Advanced Search

### Admin user?

- [ ] Yes

### Issue Description

Failed Instance Error: 
select-name: Ensures select element has an accessible name

Snippet:
`<select name="op" id="op" class="input-small"><option value="AND">all</option> <option value="OR">any</option></select>`

### Suggested Fix

Revise the select name to something more descriptive. Possibly "Select all or any"
